docs/topics/snippets.md

```diff
@@ -575,7 +575,7 @@ Similar URL customisations are also possible for the snippet chooser views throu
 
 The {attr}`~wagtail.snippets.views.snippets.SnippetViewSet.list_display` attribute can be set to specify the columns shown on the listing view. To customise the number of items to be displayed per page, you can set the {attr}`~wagtail.snippets.views.snippets.SnippetViewSet.list_per_page` attribute (or {attr}`~wagtail.snippets.views.snippets.SnippetViewSet.chooser_per_page` for the chooser listing).
 
-To customise the base queryset for the listing view, you could override the {meth}`~wagtail.snippets.views.snippets.SnippetViewSet.get_queryset` method.
+To customise the base queryset for the listing view, you could override the {meth}`~wagtail.snippets.views.snippets.SnippetViewSet.get_queryset` method. Additionally, the {attr}`~wagtail.snippets.views.snippets.SnippetViewSet.ordering` attribute or the {meth}`~wagtail.snippets.views.snippets.SnippetViewSet.get_ordering` method can be used to specify the default ordering of the listing view.
```

Comment:
The mention of `get_ordering` should be removed here - will do that when I merge.